### PR TITLE
accesskey always on for non windows

### PR DESCRIPTION
**Pull request by jinek**

### CODE REVIEW

File: src/Consolonia.Core/Infrastructure/ConsoleWindow.cs [hunk 1]
```diff
@@ -39,6 +44,17 @@ public ConsoleWindow()
             Console.FocusEvent += ConsoleOnFocusEvent;
             Handle = null!;
             PixelBuffer = new PixelBuffer(Console.Size);
+            _accessKeysAlwaysOn = !Console.SupportsAltSolo;
+            if(_accessKeysAlwaysOn)
+                _accessKeysAlwaysOnDisposable = AccessText.ShowAccessKeyProperty.Changed.SubscribeAction(OnShowAccessKeyPropertyChanged);
+        }
```
[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Core/Infrastructure/ConsoleWindow.cs]: I have fixed this. Can you check and mark this comment as resolved?

[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Core/Infrastructure/ConsoleWindow.cs]: It was you who opened this topic. Please mark it as resolved yourself.

[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Core/Infrastructure/ConsoleWindow.cs]: no, you haven't, It's still open

[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Core/Infrastructure/ConsoleWindow.cs]: Then accept a feedback for your developers please. Once the issue has been solved you should be able to mark it as resolved yourself.

[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Core/Infrastructure/ConsoleWindow.cs]: Give me a link please. I want to track the status of this suggestion

[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Core/Infrastructure/ConsoleWindow.cs]: Yes please, if you are talking about your repository.

**Review by tomlm — COMMENTED** [review]
Why disable? I notice that JetBrians rider (Which I believe is an avlonia app) responds to Alt keys. Is there a technical limitation?

**Comment by jinek** [comment]
> Why disable? I notice that JetBrians rider (Which I believe is an avlonia app) responds to Alt keys. Is there a technical limitation?

`System.Console` does not process ALT key. Same most Linux terminals. So for this cases access keys will be always visible.
Rider is Java application

File: src/Consolonia.Core/Infrastructure/ConsoleWindow.cs [hunk 2]
```diff
@@ -296,6 +306,14 @@ public void Dispose()
             GC.SuppressFinalize(this);
         }
 
+        private void OnShowAccessKeyPropertyChanged(AvaloniaPropertyChangedEventArgs<bool> args)
+        {
+            if (args.Sender != _inputRoot) return;
+            if (args.GetNewValue<bool>()) return;
+
+            _inputRoot.ShowAccessKeys = true;
+        }
```
[the remark this answers is not in the document]
Comment (jinek) [reply src/Consolonia.Core/Infrastructure/ConsoleWindow.cs:315]: What are you talking about??? All UI stuff goes on ui thread.

**Comment by tomlm** [comment]
When I tried the branch on ubuntu it didn't display any accelerators...it wasn't clear if you were trying to disable showing the accelerators or trying to turn them on permentaly.

**Review by tomlm — APPROVED** [review]

**Comment by jinek** [comment]
Let me investigate further then

**Comment by jinek** [comment]
> When I tried the branch on ubuntu it didn't display any accelerators...it wasn't clear if you were trying to disable showing the accelerators or trying to turn them on permentaly.

default ubuntu terminal? Could you please check if `CursesConsole` is instantiated in your scenario?

**Comment by tomlm** [comment]
it's good. I didn't notice there was a -linux and -linux2 branch, I was looking at the wrong one.

**Pull request merged**